docs/dg/dev/architecture/module-api/semantic-versioning-zero-major-version.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Semantic Versioning - Zero-Major Version
+description: Spryker releases early as zero-major. Learn more about them and their connection to the Early Access features in this document.
+last_updated: Apr 9, 2025
+template: concept-topic-template
+redirect_from:
+  - /docs/scos/dev/architecture/module-api/module-api.html
+related:
+  - title: Semantic versioning - major vs. minor vs. patch release
+    link: docs/scos/dev/architecture/module-api/semantic-versioning-major-vs.-minor-vs.-patch-release.html
+---
+
+
+Spryker adheres to [semantic versioning](https://semver.org/) to communicate the stability and expected impact of changes within its modules. As part of this versioning scheme, any module with a version *below 1.0.0* is considered a *zero-major* module.
+
+This document describes what using such a module means for *developers* and *project stakeholders*, focusing on technical expectations, functional completeness, and versioning behavior.
+
+## Zero-major module definition
+
+Zero-major modules are *early-stage implementations* that are publicly released for evaluation and early integration. These modules follow a versioning pattern of `0.x.x` and are not considered stable.
+
+A module remains zero-major until it reaches version *1.0.0*, at which point it is considered **stable** and the full set of [semantic versioning guarantees](https://docs.spryker.com/docs/dg/dev/architecture/module-api/semantic-versioning-major-vs.-minor-vs.-patch-release.html) apply.
+
+## Technical characteristics of zero-major modules
+
+- ❌ Backward compatibility is not guaranteed: breaking changes may occur between `0.x.x` versions and even after reaching `1.0.0`  
+- ❌ Migration effort is unpredictable: updates may introduce significant migration requirements  
+- ✅ Code must be production-level and free of known bugs  
```

Comment:
1. Do you mean that the code we provide must be production level?
2. What are known bugs in this context?

Reply:
These are very excellent questions!
1. Yes. But i think we should maybe rephrase it to be more safe, something like: the code (that we develop), must work according to designed requirements without any known bugs.
2. Maybe we could say, critical bugs? or workflow breaking bugs? or something. The idea is, that if we are aware of any bug, that does not allow the feature to be used, that should be fixed beforehand.